build-support/bin/generate_travis_yml.py

```diff
@@ -303,11 +303,16 @@ def linux_fuse_shard() -> Dict:
   }
 
 
-def _osx_env_with_pyenv(python_version: PythonVersion) -> List[str]:
+def _osx_env() -> List[str]:
   return [
     'PATH="/usr/local/opt/openssl/bin:${PATH}"',
     'LDFLAGS="-L/usr/local/opt/openssl/lib"',
     'CPPFLAGS="-I/usr/local/opt/openssl/include"',
+  ]
+
+
+def _osx_env_with_pyenv(python_version: PythonVersion) -> List[str]:
+  return _osx_env() + [
```

Comment:
Nitty suggestion: we've recently been using this style more and I think it's quite nice:

```python
return [
  *osx_env(),
  "PATH=...",
]